LCLSGeneral/LCLSGeneral/LCLSGeneral.plcproj

```diff
@@ -17,7 +17,7 @@
     <CombineIds>false</CombineIds>
     <Company>SLAC</Company>
     <Released>false</Released>
-    <Title>LCLS General</Title>
+    <Title>LCLS General-nrw</Title>
```

Comment:
Reminder to check this again before merge

Reply:
done